pom.xml

```diff
@@ -258,14 +258,12 @@
       </snapshots>
     </repository>
     <repository>
-      <id>central_without_mirror</id>
+      <id>google-maven-central</id>
```

Comment:
Yeah, this is more like what I was thinking. But does it resolve the hard-coded SBT issue? If so, OK.

Reply:
Yes, just for clarification, https://github.com/apache/spark/commit/c992716a33d78ce0a9aa78b26f5bdb45c26c2a01 fixed both hard-coded SBT issue and Github Action issue (SPARK-30572, SPARK-30534) given my tests. This is just a different way to fix both.